addons/website_sale_product_configurator/views/website_sale_product_configurator_templates.xml

```diff
@@ -4,6 +4,21 @@
         <xpath expr="//th[hasclass('td-qty')]/span[hasclass('label')]" position="attributes">
             <attribute name='t-if'>not request.is_frontend or (request.is_frontend and is_view_active('website_sale.product_quantity'))</attribute>
         </xpath>
+        <xpath expr="//td[hasclass('td-price')]" position="inside">
+            <div
+                t-if="combination_info.get('compare_list_price') and
+                    combination_info['compare_list_price'] &gt; combination_info['price']"
+                t-attf-class="text-danger oe_striked_price"
+                t-out="combination_info['compare_list_price']"
+                t-options='{
+                    "widget": "monetary",
+                    "display_currency": (pricelist or product).currency_id,
+                }'
+            />
+        </xpath>
+        <xpath expr="//td[hasclass('td-price')]//div" position="attributes">
```

Comment:
I tried using this, but I encountered an error. I found out that `hasclass` doesn't work with `t-attf-class`. Therefore, I will assign a name attribute to the specific div in the main template and utilize it here to ensure clarity in reference.

Reply:
Hmm, I wonder whether that could break the template in case the user only updates the eCommerce configurator module, but not the overridden sale module :thinking:

Reply:
It works like this - `div[contains(@t-attf-class,'oe_striked_price')`. I will remove the names form the main templates too.